src/main/java/mate.academy/Main.java

```diff
@@ -1,17 +0,0 @@
-package mate.academy;
-
-import java.util.List;
-import mate.academy.lib.Injector;
-import mate.academy.model.Product;
-import mate.academy.service.ProductService;
-
-public class Main {
```

Comment:
You should not remove the Main class, it's the starting oint of your application.